pkg/api/config_test.go

```diff
@@ -82,7 +99,24 @@ func TestValidate(t *testing.T) {
 				InputConfiguration: InputConfiguration{},
 			},
 			expectedValid: false,
-			expectedError: "no build_root or test_base_image has been set",
+			expectedError: "both build_root and test_base_image cannot be empty",
+		},
+		{
+			id: "base images",
+			config: ReleaseBuildConfiguration{
+				InputConfiguration: InputConfiguration{
+					BuildRootImage: &BuildRootImageConfiguration{
+						ImageStreamTagReference: &ImageStreamTagReference{
+							Tag: "test",
+						},
+					},
+					BaseImages: map[string]ImageStreamTagReference{"test": {Cluster: "test"},
+						"test2": {Tag: "test2"}, "test3": {Cluster: "test3"},
+					},
+				},
+			},
+			expectedValid: false,
+			expectedError: "[[base_images.test.cluster: invalid URL given: parse test: invalid URI for request, base_images.test.tag: value required but not provided], [base_images.test3.cluster: invalid URL given: parse test3: invalid URI for request, base_images.test3.tag: value required but not provided]]",
```

Comment:
See @stevekuznetsov's comment on removing the check for the specific error message on [my PR](https://github.com/openshift/ci-operator/pull/125#discussion_r218523080).